From 9465df7db705a3d94f801e247537cf030727567f Mon Sep 17 00:00:00 2001
From: Jason Liu <r64343@freescale.com>
Date: Wed, 11 Apr 2012 17:16:36 +0800
Subject: [PATCH 1300/2463] ENGR00179426-1 i.mx6: iomux: NO_PAD_I/NO_PAD_MUX
 not defined

NO_PAD_I/NO_PAD_MUX not defined, which will cause build error
According to iomux-v3.h, the NO_PAD_I/NO_PAD_MUX should be 0
for the pins which does not have PAD/MUX config.

Signed-off-by: Jason Liu <r64343@freescale.com>
---
 arch/arm/plat-mxc/include/mach/iomux-mx6q.h |    2 --
 arch/arm/plat-mxc/include/mach/iomux-v3.h   |    2 ++
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/plat-mxc/include/mach/iomux-mx6q.h b/arch/arm/plat-mxc/include/mach/iomux-mx6q.h
index ec6b228..9944067 100644
--- a/arch/arm/plat-mxc/include/mach/iomux-mx6q.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-mx6q.h
@@ -39,8 +39,6 @@ typedef enum iomux_config {
 	IOMUX_CONFIG_GPIO,      /* added to help user use GPIO mode */
 	} iomux_pin_cfg_t;
 
-#define NON_MUX_I       0x3FF
-#define NON_PAD_I       0x7FF
 #define MX6Q_CCM_CLK0_PAD_CTRL	(PAD_CTL_PKE | PAD_CTL_PUE |	\
 		PAD_CTL_PUS_47K_UP | PAD_CTL_SPEED_LOW |	\
 		PAD_CTL_DSE_80ohm | PAD_CTL_SRE_FAST | PAD_CTL_HYS)
diff --git a/arch/arm/plat-mxc/include/mach/iomux-v3.h b/arch/arm/plat-mxc/include/mach/iomux-v3.h
index 60fb82e..1ff80f9 100644
--- a/arch/arm/plat-mxc/include/mach/iomux-v3.h
+++ b/arch/arm/plat-mxc/include/mach/iomux-v3.h
@@ -86,6 +86,8 @@ typedef u64 iomux_v3_cfg_t;
  * Use to set PAD control
  */
 #define NO_PAD_CTRL			(1 << 17)
+#define NO_PAD_I			0
+#define NO_MUX_I			0
 #ifdef CONFIG_SOC_IMX6Q
 #define PAD_CTL_HYS			(1 << 16)
 
-- 
1.7.7.4

